app/assets/stylesheets/shared_styles/elements/radio_buttons.scss

```diff
@@ -0,0 +1,83 @@
+// scss-lint:disable SelectorDepth QualifyingElement
+
+:root {
+  --sci-radio-size: 16px;
+}
+
+input[type="radio"].sci-radio {
+  cursor: pointer;
+  flex-shrink: 0;
+  height: var(--sci-radio-size);
+  margin: 0;
+  opacity: 0;
+  position: relative;
+  width: var(--sci-radio-size);
+  z-index: 2;
+
+  + .sci-radio-label {
+    display: inline-block;
+    flex-shrink: 0;
+    height: var(--sci-radio-size);
+    margin-left: calc((var(--sci-radio-size) * -1) - 3px);
+    position: relative;
+    width: var(--sci-radio-size);
+
+    &::before {
+      animation-timing-function: $timing-function-sharp;
+      border: $border-default;
+      border-radius: 50%;
+      color: $color-white;
+      content: "";
+      font-family: "Font Awesome 5 Free";
+      font-size: calc(var(--sci-radio-size) - var(--sci-radio-size) * .375);
+      font-weight: 501;
```

Comment:
Add this to constants. You use it multiple times.

Reply:
fixed